src/main/java/ecdar/abstractions/Query.java

```diff
@@ -60,7 +59,7 @@ public ObjectProperty<QueryState> queryStateProperty() {
     }
 
     public String getQuery() {
-        return query.get();
+        return this.query.get().replace("\u2264","<=").replace("\u2265",">=");
```

Comment:
I kan bruge funktionen her

Reply:
den her replacer omvendt så nej

Reply:
I see, når i sender det afsted til Reveaal er det så med unicode characters?

Reply:
Nej vi replacer tilbage tilbage til >= og det for at kunne sende det - det ville derfor være smartere at backenden bare kunne tage i mod unicode

Reply:
Hvorfor replacer i ikke bare unicode characters når i sætter querien?

Reply:
vi replacer hver gang der skrive >= og <= i query tekstfeltet

ved ikke hvad Ulen betyder

Reply:
UIen = GUI'en
Jeg forslår at representationen af queries bliver beholdt som det er nu, men når at en Query bliver præsenteret på GUI'en bliver <= udskiftet med ≤ og >= bliver udskiftet med ≥

Reply:
vi laver en static class til at have en static metode til at replace fra >= til unicode og bruger den de steder hvor vi lige nu bare replacer - er det fint, eller var der andet også vi skulle ændre? :)

Reply:
Måske også lave den modsatte funktion så er de samlet :)